packages/web-service/src/__tests__/additional-page-data.spec.js

```diff
@@ -63,6 +63,11 @@ describe('additional page data', () => {
         methodExperience: '/enter-methods',
         previousLicence: '/previous-licence',
         register: '/register',
+        siteGridReference: '/site-grid-ref',
+        siteMap: '/upload-map',
+        siteMapThree: '/upload-map-3',
+        siteMapTwo: '/upload-map-2',
```

Comment:
`upload-map-2`, `upload-map-2` - can we rename it to something more descriptive?

Reply:
Please make sure the `CONFIRMED_NOT_COMPLETE` handler works too :) Just had a quick Slack call we need to go back to CYA page, if you complete the flow but don't submit the answers